release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_Microsoft.VisualBasic.ApplicationServices.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft.VisualBasic.ApplicationServices
+
+``` diff
+ namespace Microsoft.VisualBasic.ApplicationServices {
++    public class ApplyApplicationDefaultsEventArgs : EventArgs {
++        public Font Font { get; set; }
++        public HighDpiMode HighDpiMode { get; set; }
++        public int MinimumSplashScreenDisplayTime { get; set; }
++    }
++    public delegate void ApplyApplicationDefaultsEventHandler(object sender, ApplyApplicationDefaultsEventArgs e);
+     public class WindowsFormsApplicationBase : ConsoleApplicationBase {
++        protected HighDpiMode HighDpiMode { get; set; }
++        public event ApplyApplicationDefaultsEventHandler ApplyApplicationDefaults;
+     }
+ }
+```
+
```

Comment:
@KlausLoeffelmann look good to you?

Reply:
For the diffs, yes. Also @KathleenDollard.